src/shared/lib/time.test.ts

```diff
@@ -1,14 +1,23 @@
 import { expect, test } from 'vitest';
-import { formatUTC } from '@/shared/lib/time';
+import { getLocalTime, getUTCTime } from '@/shared/lib/time';
 import { mount } from '@vue/test-utils';
 import { i18n } from '@/shared/lib/localization';
 import TimeStamp from '@/shared/ui/components/TimeStamp.vue';
 
+// Those tests are written for GMT+3 timezone.
+// In other timezones they won't pass
```

Comment:
Set some fixed timezone for tests according to https://github.com/vitest-dev/vitest/issues/1575#issuecomment-1171085461

Reply:
Great, I didn't know about it before! I decided to do the following https://github.com/vitest-dev/vitest/issues/1575#issuecomment-1439286286